add(completion)(v7.9): add tab completion description (#788)
cap-js/docs: get-started/jumpstart.md

cds #> run the installed CLI
```

> #### 2.1. Activate CDS Shell Completion

Reviewer: ```suggestion
Activate CDS Shell Completion:
```

> The `cds` command supports starting with version `7.9` shell completion with the `tab` key for several shells and operating systems.

Reviewer: ```suggestion
The `cds` command supports shell completion with <kbd>tab</kbd> key for several shells and operating systems.
```

> For Linux, macOS and Windows use the following command to activate the shell completion.

Reviewer 2: ```suggestion
For Linux, macOS and Windows use the following command to activate the shell completion:
```
